casejs/case
# Add ENUM as property type for dynamic entity

## Description

This PR is adding the enum type to the properties accepted in dynamic entity.

## Specification

Spec : 
Enum is a new type of property
It should have two possible display _label_ and _progress bar_
The label is the default way of display 
User can't customize colors of tag label (edit from return on PR)
The input associated is the select input (thats already exists)


## How can it be tested ?

Go to http://localhost:4200/dynamic/cats/create 

Create a new cat with the properties species and status. 

![Capture d’écran 2023-07-26 à 16 40 16](https://github.com/casejs/case/assets/50170060/b8fba31d-a4ab-4771-80a7-8c5fc44c0a81)


Go to http://localhost:4200/dynamic/cats 

Check if the cats have the two yields species and status. 

![Capture d’écran 2023-07-26 à 16 40 05](https://github.com/casejs/case/assets/50170060/1d66fecb-6b15-4a41-86cb-fe4471a2cf32)



After serving the doc in your project : 

Go to http://localhost:3000/#/entities?id=enum 

Follow the doc in entity to add the enum in an other entity. 

## Check list before submitting
